cmd/jobsShow.go

```diff
@@ -79,10 +79,9 @@ func init() {
 // handles the list command
 // dispatches the list order to the transfer engine
 func HandleShowCommand(listRequest common.ListRequest) error {
-	rpcCmd := common.ERpcCmd.None()
 	if listRequest.OfStatus == "" {
 		resp := common.ListJobSummaryResponse{}
-		rpcCmd = common.ERpcCmd.ListJobSummary()
+		rpcCmd := common.ERpcCmd.ListJobSummary()
```

Comment:
given the creation of rpcCmd immediately precedes the use of it, we should drop the variable all together, or look into reducing code dupe here.

Reply:
I think I prefer it as is now since it helps debuggability if necessary